src/osd/scrubber/ScrubStore.cc

```diff
@@ -3,11 +3,14 @@
 
 #include "./ScrubStore.h"
 #include "osd/osd_types.h"
+#include "common/debug.h"
 #include "common/scrub_types.h"
 #include "include/rados/rados_types.hpp"
 
 #include "pg_scrubber.h"
 
+#include <sstream>
```

Comment:
seems to be added because of the ostringstream on the next line, but I don't see that one being used.
probably should remove line 11(14) instead